packages/aws-cdk-lib/aws-apigatewayv2-integrations/lib/http/sqs.ts

```diff
@@ -0,0 +1,117 @@
+import * as apigwv2 from '../../../aws-apigatewayv2';
+import * as iam from '../../../aws-iam';
+import * as sqs from '../../../aws-sqs';
+
+/**
+ * Properties to initialize `HttpSqsIntegration`.
+ */
+export interface HttpSqsIntegrationProps {
+  /**
+   * Specifies how to transform HTTP requests before sending them to the backend.
+   *
+   * When the subtype is either `START_EXECUTION` or `START_SYNC_EXECUTION`,
+   * it is necessary to specify the `StateMachineArn`.
+   * Conversely, when the subtype is `STOP_EXECUTION`, the `ExecutionArn` must be specified.
+   *
+   * @see https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-parameter-mapping.html
+   *
+   * @default - specify only `StateMachineArn`
```

Comment:
Below in the code you're using `this.props.queue.queueUrl` so this doesn't seem accurate.

Reply:
I have fixed it.